certified-connectors/DocuSignDemo/apiDefinition.swagger.json

```diff
@@ -1005,6 +1005,84 @@
         },
         "deprecated": false,
         "x-ms-visibility": "internal"
+      },
+      "put": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```

Comment:
@sriharip-docusign  Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

Reply:
@sriyen-msft 
I have updated the PR to add schema to parse the JSON response.